editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/ContactForm.tsx

```diff
@@ -44,14 +60,40 @@ export default function ContactForm({ formikConfig, onSuccess }: FormProps) {
   const onChangeFn = (type: string, event: ChangeEvent<HTMLInputElement>) =>
     formik.setFieldValue(type, event.target.value);
 
+  const sendEmailChangeSlackNotification = async (newEmail: string) => {
+    const skipTeamSlugs = [
+      "open-digital-planning",
+      "opensystemslab",
+      "planx",
+      "templates",
+      "testing",
+      "wikihouse",
+    ];
+    if (skipTeamSlugs.includes(teamSlug)) return;
+
+    const message = `:e-mail: *${teamSlug}* updated their Gov UK Notify reply-to email to *${newEmail}*; ensure this is configured in Gov UK Notify dashboard.`;
+
+    return axios.post(
+      `${import.meta.env.VITE_APP_API_URL}/send-slack-notification`,
```

Comment:
A minor limitation of this endpoint is that is currently supports a single webhook URL / channel. 

Therefore we'll be logging this to #planx-notifications - this should only ever be council email addresses, never personal users details so I think that's fine for a shared ODP Channel, but in the future we may want to consider supporting a simple split between #planx-notifications & #planx-notifications-internal ?

Reply:
Thanks for pointing this out, I could easily see this API endpoint taking a channel argument in future. I think this is great for now though.